pages/filter.tsx

```diff
@@ -131,6 +163,9 @@ export const FilterPage: NextPage = () => {
           })}
         </CheckboxGroup>
       </FilterSection>
+      <Link href={`/`} passHref>
```

Comment:
I'm still of the opinion that the filter choices should be stored in the URL :-)